migrations/1755621780124-addressBookItemLength.ts

```diff
@@ -0,0 +1,17 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
```

Comment:
On localdev I get a lint warning here since these are only used as types.

Reply:
I'm running yarn lint and it is fine for me?